Gems/CsvSpawner/Code/Source/CsvSpawner/CsvSpawnerUtils.cpp

```diff
@@ -234,6 +239,11 @@ namespace CsvSpawner::CsvSpawnerUtils
             AZ::Transform transform = parentTransform * entityConfig.m_transform *
                 GetRandomTransform(spawnConfig.m_positionStdDev, spawnConfig.m_rotationStdDev, spawnConfig.m_scaleStdDev, gen);
 
+            const float scale = transform.GetUniformScale();
+            if (AZ::IsClose(scale, 0.0f, MinimalScale))
```

Comment:
- Minimum = the lowest possible (often exact, measurable).
- Minimal = very small (general, approximate, qualitative).

```suggestion
            if (AZ::IsClose(scale, 0.0f, MinimumScale))
```